pyproject.toml

```
@@ -25,13 +25,13 @@ requires-python = "~=3.9"
 dynamic = ["version"]
 
 dependencies = [
-    "oteapi-core >=0.5.2,<1",
+    "oteapi-core >=0.6.0,<1",
```

Keep below v0.6 until support for pydantic v2 has been implemented.
```suggestion
    "oteapi-core >=0.5.2,<0.6",
```